stable/elasticsearch/values.yaml

```diff
@@ -36,6 +36,9 @@ initImage:
   pullPolicy: "Always"
 
 cluster:
+  # If you want a minimal, single-node Elasticsearch cluster, disable the following parameter
```

Comment:
Can you be a bit more explicit about what this needs to be set to to deploy a single node cluster?
Also make a note at the `data.replicas` variable that it needs to be set to 1 for a single node cluster.

Reply:
It may even be better to overwrite data.replicas to be 1 whatever is in values.yaml, and be clear about it.

Reply:
That is definitely a good idea. :)